test/internal/utils.go

```diff
@@ -94,13 +93,16 @@ func (tc TestContext) AddPackagemanifestsTarget() error {
 	return nil
 }
 
-// ReplaceInFile replaces all instances of old with new in the file at path.
-func ReplaceInFile(path, old, new string) {
+// ReplaceRegexInFile finds all strings that match `match` and replaces them
+// with `replace` in the file at path.
+func ReplaceRegexInFile(path, match, replace string) {
+	matcher, err := regexp.Compile(match)
+	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 	info, err := os.Stat(path)
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 	b, err := ioutil.ReadFile(path)
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
-	s := strings.Replace(string(b), old, new, -1)
+	s := matcher.ReplaceAllString(string(b), replace)
 	err = ioutil.WriteFile(path, []byte(s), info.Mode())
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
```

Comment:
we also should return an error when the content is not found to avoid this kind of scenario. 
However, we can do it as follow up.

Reply:
I went ahead and added a check to make sure a replacement actually occurs.